src/Controls/src/Core/Application/Application.cs

```diff
@@ -220,8 +220,9 @@ public static Color? AccentColor
 
 			return null;
 #elif ANDROID
-			if (Current?.Windows?.Count > 0)
-				return Current.Windows[0].MauiContext.Context?.GetAccentColor();
+			var context = (Current?.Windows?.Count > 0 && Current.Windows[0].MauiContext is not null) ? Current.Windows[0].MauiContext.Context : Current?.FindMauiContext()?.Context;
```

Comment:
Did you validate if `Current?.FindMauiContext()?.Context` will return an accent color? That's the application context on the Activity so I feel like it won't have the theme resources loaded into it.